src/Bicep.Cli/Commands/RootCommand.cs

```diff
@@ -182,10 +180,9 @@ private void PrintHelp()
     Examples:
       bicep build-params params.bicepparam
       bicep build-params params.bicepparam --stdout
-      bicep build-params params.bicepparam --outfile otherParams.json
+      bicep build-params params.bicepparam --outfile-params otherParams.json --outfile-bicep otherMain.json
```

Comment:
I think you intended to update the [`generate-params`](https://github.com/Azure/bicep/blob/25991d68d51fd808a49c0c7ee54562ac59cfa0f9/src/Bicep.Cli/Commands/RootCommand.cs#L119)  command help here instead. 

Also for `build-params` command (that compiles the bicepparam file), we decided to only have parameters file output (even though we can also produce bicep output) so only `--outfile` should be an option here

Reply:
yeah, it seems I mixed up the `params` commands 🤦‍♂️ I created a commit, it should be fixed now.